internal/services/cognitive/cognitive_deployment_resource.go

```diff
@@ -366,6 +379,11 @@ func (r CognitiveDeploymentResource) Read() sdk.ResourceFunc {
 				if v := properties.VersionUpgradeOption; v != nil {
 					state.VersionUpgradeOption = string(*v)
 				}
+
+				if v := properties.DynamicThrottlingEnabled; v != nil {
+					state.DynamicThrottlingEnabled = *v
+				}
```

Comment:
This can be simplified to 
```suggestion
				state.DynamicThrottlingEnabled := pointer.From(properties.DynamicThrottlingEnabled)
```